ERCS/erc-7744.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7744
+title: Code Index
+description: Indexing smart contract bytecode on Ethereum.
```

Comment:
```suggestion
title: Code Index
description: Indexing smart contract bytecode
```

In addition to removing the fluff, you should expand a little more on what this proposal proposes. What is a code index? Who would want it? Try not to just restate your title in your description. Instead, elaborate in the description on the ideas introduced in the title.

Reply:
Elaborated description. Not sure if there is any length requirement on this field. Im concerned if it's too long now?